spec/models/account/user_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Account::User, type: :model do
+  context 'validation tests' do
+    let(:user) { build(:user) }
+
+    it 'is valid with valid attributes' do
+      expect(user.save).to eq(true)
+    end
+    
+    it "is not valid without a first name" do
+      user.first_name = nil
+      expect(user.save).to_not eq(true)
```

Comment:
this example is not quite accurate.. it's not showing the reason why it's not saved
may be different reasons.. try to have each test example to bring some explicit logic about the test subject